README.md

```diff
@@ -0,0 +1,96 @@
+# Vellum Ruby Library
+
+[![fern shield](https://img.shields.io/badge/%F0%9F%8C%BF-SDK%20generated%20by%20Fern-brightgreen)](https://github.com/fern-api/fern)
+
+
+The Vellum Ruby Library provides access to the Vellum API from Ruby.
+
+
+## API Docs
+You can find Vellum's complete API docs at [docs.vellum.ai](https://docs.vellum.ai).
+
+## Installation
+
+Install the gem and add to the application's Gemfile by executing:
+
+```sh
+bundle add vellum_ai
+```
+
+If bundler is not being used to manage dependencies, install the gem by executing:
+
+```sh
+gem install vellum_ai
+```
+
+## Usage
+
+```ruby
+require "vellum_ai"
+
+
+client = Vellum::Client.new(api_key: "YOUR_API_KEY")
+
+result = client.generate(
+    deployment_name: "my-deployment",
+    requests=[{
+        input_values: [{"question": "Can I get a refund?"}]
+    }]
+)
+
+puts result.text
+```
+
+## Async Client
+
+```ruby
+require "vellum_ai"
+
+
+client = Vellum::AsyncClient.new(api_key: "YOUR_API_KEY")
+
+result = client.generate(
```

Comment:
We'll want execute_prompt here and above instead - will push an edit here in a few